README.md

```diff
@@ -19,14 +19,16 @@ Rhino is configured using the following environment variables:
 - `DATABASE_URL`: how to connect to the MYSQL database. Form: `jdbc:mysql://HOSTNAME:3306/DATABASE?user=USERNAME&password=PASSWORD`
 - `CORPUS_BUCKET`: name of the corpus bucket
 - `EDITORIAL_BUCKET`: optional, name of the editorial bucket
-- `CONTENT_REPO_URL`: URL for the contentrepo instance to use, e.g. `http://localhost:8002/v1/`
 - `TAXONOMY_URL`: URL of the access innovations server to use, e.g. `https://localhost:9138/servlet/dh`
 - `THESAURUS`: Name of the thesaurus to use, e.g. `plosthes.2017-2`
 - `LOG_LEVEL`: optional, default is `warn`. the level of logging, e.g. `debug`
+- `AWS_ACCESS_KEY_ID`, `AWS_SECRET_KEY`: AWS keypair to use
+- `AWS_REGION`: AWS region to use
+- `AWS_ROLE_ARN`: The ARN of the AWS role to assume
 
 Full example of running using `cargo`:
 ```
-$ CORPUS_BUCKET=mogilefs-prod-repo CONTENT_REPO_URL=http://localhost:8002/v1/ TAXONOMY_URL=https://localhost:9138/servlet/dh THESAURUS=plosthes.2017-2 DATABASE_URL="jdbc:mysql://localhost:3306/ambra?user=root&password=password" mvn package cargo:run`
+$ AWS_SECRET_KEY=XXX AWS_ACCESS_KEY_ID=AKIA4SPCIUSTALRESEYQ AWS_AWS_REGION=us-east-1 AWS_ROLE_ARN="arn:aws:iam::429579970117:role/PLOS_DevAccess" CORPUS_BUCKET=my-bucket TAXONOMY_URL=https://plos.accessinn.com:9138/servlet/dh THESAURUS=plosthes.2017-2 DATABASE_URL="jdbc:mysql://localhost:3306/ambra?user=root&password=password" mvn package cargo:run
```

Comment:
I think this is good for now but at some point we may want to make the creds optional as the security context of the process could be set elsewhere (like a machine role or in the case of fargate, the task role).

Reply:
I am using the AWS java sdk default authentication, except for the AWS role, which I had to set myself. So as far as I know, this should be possible.

The goal somewhat conflicts with the request to get an error when we fail to provide an env var, though. I lean towards throwing an error at startup when we fail to provide necessary env vars, as we plan to config the system using env vars at the moment.

Reply:
Agreed this is the way to go for now.

Reply:
a87c722